### PR TITLE
Feature npt

### PR DESCRIPTION
**Pull request by ianmkenney**
- Use NPT for FEP #27 
- Use Parrinello-Rahman in MD_NPT #28

### CODE REVIEW

**Comment by orbeckst** [comment]
@ianmkenney can you confirm that this is working?

File: mdpow/templates/fep_opls.mdp [hunk 1]
```diff
@@ -84,7 +84,11 @@ Tcoupl                   = no
 tau_t                    = 0.1
 tc-grps                  = System
 ref_t                    = 300
-Pcoupl                   = no
+Pcoupl                   = parrinello-rahman
+Pcoupltype		 = isotropic
+tau_p			 = 1.0
+compressibility		 = .000045
```
Comment (orbeckst) [inline mdpow/templates/fep_opls.mdp]: should be 4.6e-5 (not 4.5e-5)

File: mdpow/templates/equil_opls.mdp [hunk 2]
```diff
@@ -0,0 +1,102 @@
+; Gromacs mdp file
```
Comment (orbeckst) [inline mdpow/templates/equil_opls.mdp]: Where do we use this file?

File: mdpow/templates/bar_opls.mdp [hunk 3]
```diff
@@ -84,7 +84,11 @@ Tcoupl                   = no
 tau_t                    = 0.1
 tc-grps                  = System
 ref_t                    = 300.0
-Pcoupl                   = no
+Pcoupl                   = Parrinello-Rahman
+Pcoupltype               = isotropic
+compressibility          = 0.000045
```
Comment (orbeckst) [inline mdpow/templates/bar_opls.mdp]: should be 4.6e-5

**Comment by ianmkenney** [comment]
I checked out my simulations from this branch and the correct parameters were used (ie Parrinello-Rahman).

**Comment by orbeckst** [comment]
Ok, together with the small fixes that you're committing, I can merge this.

**Pull request merged**